examples/runners.jl

```diff
@@ -0,0 +1,87 @@
+# # Runners
```

Comment:
I'd like this to be renamed to "Mountain Runners" or something along those lines so that it is a bit less vague.